packages/dashboard/src/app/views/shared/savedTemplateGridView.js

```diff
@@ -64,7 +54,7 @@ const StoryGrid = styled(CardGrid)`
 
 const SavedTemplateGridView = ({
   templates,
-  centerActionLabelByStatus,
+  // centerActionLabelByStatus,
   // bottomActionLabel,
```

Comment:
Should we remove this commented out code?

Reply:
Yes but not here: https://github.com/google/web-stories-wp/issues/8746